web/react/components/channel_members.jsx

```diff
@@ -137,6 +137,7 @@ module.exports = React.createClass({
                                     memberList={this.state.member_list}
                                     isAdmin={isAdmin}
                                     handleRemove={this.handleRemove}
+                                    allAdminAccess={true}
```

Comment:
Not saying you're wrong, but why are we adding this prop here instead of just directly to member_list?